components/hackerone/hackerone.app.mjs

```diff
@@ -1,11 +1,96 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
+
 export default {
   type: "app",
   app: "hackerone",
-  propDefinitions: {},
+  propDefinitions: {
+    name: {
+      type: "string",
+      label: "Name",
+      description: "Name of the organization",
+    },
+    email: {
+      type: "string",
+      label: "Email",
+      description: "Email of the invitee",
+    },
+    organizationAdmin: {
+      type: "string",
+      label: "Organization Admin",
+      description: "Sets the invitee as an organization admin",
+    },
+    permissions: {
+      type: "string[]",
+      label: "Permissions",
+      description: "The permissions added to the new organization group",
+      options: constants.PERMISSIONS,
+    },
+    organizationId: {
+      type: "string",
+      label: "Organization ID",
+      description: "ID of the organization",
+      async options() {
+        const { data: organizationsIDs } = await this.getOrganizations();
+
+        return organizationsIDs.map(({ id }) => ({
+          value: id,
+        }));
+      },
+    },
+  },
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    _baseUrl() {
+      return "https://api.hackerone.com/v1";
+    },
+    async _makeRequest(opts = {}) {
+      const {
+        $ = this,
+        path,
+        auth,
+        ...otherOpts
+      } = opts;
+      return axios($, {
+        ...otherOpts,
+        url: this._baseUrl() + path,
+        auth: {
+          ...auth,
+          username: `${this.$auth.api_token_identifier}`,
+          password: `${this.$auth.api_token}`,
+        },
+      });
+    },
+    async createInvitation({
+      organization_id, ...args
```

Comment:
Since these are variables used only within our code and the param name is not sent to the API, camelCase should be used (`organizationId`)

Please adjust all the instances of path params that are not using camelCase